test_msgs/package.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>test_msgs</name>
+  <version>0.0.0</version>
+  <description>A package containing message definitions and fixtures used exclusively for testing purposes.</description>
+  <maintainer email="karsten@osrfoundation.org">Karsten Knese</maintainer>
```

Comment:
Not sure why you picked @Karsten1987 but I don't mind :wink:

Reply:
he is the one that started the original work in https://github.com/ros2/common_interfaces/pull/30 and I took over, I figured he'd like to keep maintainership :stuck_out_tongue_winking_eye:

Reply:
> I figured he'd like to keep maintainership

Very generous of you :laughing: Works for me...